src/patterns/step-by-step-navigation/index.md.njk

```diff
@@ -8,7 +8,7 @@ backlog_issue_id: 171
 layout: layout-pane.njk
 ---
 
-The step by step navigation pattern presents an end to end journey in logical steps, providing links users out to the content that helps them complete each step.
+The step by step navigation pattern presents an end to end journey in logical steps, with links to content to help users complete each step.
```

Comment:
No problem! I suspect the sentence we're looking at was meant to be written as either:
```suggestion
The step by step navigation pattern presents an end to end journey in logical steps, providing links out to the content that helps them complete each step.
```
or
```suggestion
The step by step navigation pattern presents an end to end journey in logical steps, linking users out to the content that helps them complete each step
```
Just wondering if we thought it'd be better to stay closer to the original text? The edit proposed is fine, though it'd be good to see if we can remove the duplicate 'to's:
```suggestion
The step by step navigation pattern presents an end to end journey in logical steps, with links to content that help users complete each step.
```

Reply:
Thanks! Yeah, removing the duplicates sounds like a good idea. In our third suggestion here, should we change 'help' to 'helps' (because 'content' is singular)?

Reply:
Yes, and you're right! I'll commit it as you've suggested.